lib/new_relixir/agent.ex

```diff
@@ -117,7 +117,7 @@ defmodule NewRelixir.Agent do
   end
 
   def request(url, body \\ "[]") do
-    headers = [{'Content-Encoding', 'identity'}]
+    headers = [{"Content-Encoding", "identity"}]
```

Comment:
Fixes the expected type for `headers`
```elixir
lib/new_relixir/agent.ex:121: 
The call hackney:post(Vurl@1::any(),Vheaders@1::[{<<_:128>>,[100 | 101 | 105 | 110 | 116 | 121,...]},...],Vbody@1::any(),['with_body',...]) will never return since it differs in the 2nd argument from the success typing arguments: (binary() | [binary() | maybe_improper_list(any(),binary() | []) | char()] | {'hackney_url',atom(),atom(),binary(),'undefined' | binary(),'nil' | 'undefined' | binary(),binary(),binary(),string(),'undefined' | integer(),binary(),binary()},[{binary() | [any()],binary() | {_}}],any(),[any()])
```

Reply:
Good catch